tests/Unit/CustomerTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Unit;
+
+use Carbon\Carbon;
+use Digistorm\SchoolEasyPay\Card;
+use Digistorm\SchoolEasyPay\Config;
+use Digistorm\SchoolEasyPay\Customer;
+use Digistorm\SchoolEasyPay\Enums\NotificationMethod;
+use Digistorm\SchoolEasyPay\Enums\PaymentFrequency;
+use Digistorm\SchoolEasyPay\Enums\PaymentOption;
+use Digistorm\SchoolEasyPay\Exceptions\SchoolEasyPayException;
+use PHPUnit\Framework\TestCase;
+
+class CustomerTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->config = new Config('test', 'test', 'test', true);
+    }
+
+    /**
+     * Test that true does in fact equal true
+     */
+    public function testToJson()
+    {
+        $card = (new Card())
+            ->setCardProxy('test-card-proxy')
+        ;
+
+        $customer = (new Customer($this->config))
+            ->setCard($card)
+            ->setCustomerReference('1234')
+            ->setPaymentFrequency(PaymentFrequency::WEEKLY)
+        ;
+
+        $json = $customer->toJson();
+
+        $this->assertEquals('{"customerReference":"1234","paymentFrequency":32,"card":{"cardProxy":"test-card-proxy"}}', $json);
+    }
+
+    public function testSetAndGetCustomerReference()
+    {
+        $customer = new Customer($this->config);
+        $customerReference = 'REF12345';
+        $customer->setCustomerReference($customerReference);
+
+        $this->assertEquals($customerReference, $customer->getCustomerReference());
+    }
+
+    /**
+     * @throws SchoolEasyPayException
```

Comment:
Maybe we don't need these comments in the tests?

Reply:
I just added them to satisfy the IDE (it complains with a warning / yellow line otherwise).

Reply:
I've wrapped the relevant calls in `try` / `catch` to avoid needing the annotation.